bindings/nodejs/examples/example_s3.js

```diff
@@ -0,0 +1,38 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import { Operator } from "opendal"
+
+async function main() {
+  const op = new Operator(
+    "s3", {
+    root: "/test_opendal",
+    bucket: "your bucket name",
+    region: "your AWS region",
+    endpoint: "xxxx.s3.xxxx.amazonaws.com",
+    access_key_id: "AK*******************",
+    secret_access_key: "*****************",
```

Comment:
It's better to get the secrets from env.

Reply:
> It's better to get the secrets from env.

This example is a bit complex. Additionally, opendal will load from the environment by default. Therefore, I suggest using `your access key id` as a demonstration.

Reply:
You could take https://github.com/apache/incubator-opendal/blob/main/bindings/nodejs/benchmark/node.js as an example.

Could also write some doc to inform this tip.